production/helm/ct.yaml

```diff
@@ -1,6 +1,11 @@
+---
+remote: origin
+target-branch: main
 chart-repos:
   - elastic=https://helm.elastic.co
   - grafana=https://grafana.github.io/helm-charts
   - prometheus=https://prometheus-community.github.io/helm-charts
+  - minio=https://helm.min.io
+helm-extra-args: --timeout 600s
```

Comment:
hmm maybe this --timeout flag can be used as a new default instead?

Reply:
I'm not sure how to do that? I'm pretty sure these are passed as extra args to `helm init` when using the ct tool for CI jobs, maybe there's a config for it in the CI jobs?

Reply:
ohh got it, I thought it was a Loki timeout. nvm